modules/core/client/controllers/login.client.controller.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+angular.module('core').controller('LoginController', ['$scope',
+                /*'Authentication',*/
+              	//function ($scope, Authentication) {
+                	// This provides Authentication context.
+                	//scope.authentication = Authentication;
+    function ($scope) {
+      $scope.usernameField = "";
+      $scope.passwordField = "";
+
+      $scope.login = function(usernameField, passwordField) {
+        console.log("Pressed Submit");
```

Comment:
Pointing out that we still have this console statement in case you wanted to yank it out